frontend/src/components/HomePage/index.jsx

```diff
@@ -1,74 +1,97 @@
 import { useState, useEffect } from 'react';
 import './HomePage.css';
-
 import Card from './Card';
 import Banner from './Banner';
 import Filters from './Filters';
 
+// Define categories and associated keywords
+const categories = {
```

Comment:
This and the below should be in a new file.

Reply:
Agreed. I wanted to see that it worked conceptually and realized later I needed the categories passed from the index page into the bill page, so I just brought the same code.
Next I'll focus on bringing those categories to another file to clean it up :)